Telemetry/RulesTest.cs

```diff
@@ -25,11 +47,335 @@ public void Should_Return_OK_Status()
+            this.disposeRulesList.Add(newRule.Id);
+
+            // Act
+            newRule.Enabled = false;
+
+            request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + newRule.Id);
+            request.AddHeader("Content-Type", "application/json");
+            request.SetContent(JsonConvert.SerializeObject(newRule));
+
+            var updateResponse = this.httpClient.PutAsync(request).Result;
+            var updatedRule = JsonConvert.DeserializeObject<RuleApiModel>(updateResponse.Content);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, updateResponse.StatusCode);
+            Assert.Equal(newRule.Enabled, updatedRule.Enabled);
+        }
+
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void DeleteRuleReturnsOK_IfRuleExists()
+        {
+            string ruleId = "TESTRULEID" + DateTime.UtcNow.ToString("yyyyMMddHHmmss") + "-" + Guid.NewGuid();
+
+            // Arrange  
+            var ruleRequest = this.GetSampleRuleWithCalculation("Average", "600000");
+
+            var request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + ruleId);
+            request.AddHeader("Content-Type", "application/json");
+            request.SetContent(JsonConvert.SerializeObject(ruleRequest));
+
+            var response = this.httpClient.PutAsync(request).Result;
+
+            // Dispose after tests run
+            this.disposeRulesList.Add(ruleId);
+
+            // Act
+            request = new HttpRequest(Constants.TELEMETRY_ADDRESS + RULES_ENDPOINT_SUFFIX + "/" + ruleId);
+
+            response = this.httpClient.DeleteAsync(request).Result;
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
+        private RuleApiModel GetSampleRuleWithCalculation(string calculation, string timePeriod)
+        {
+            var condition = new ConditionApiModel()
+            {
+                Field = "pressure",
+                Operator = "GreaterThan",
+                Value = "150"
+            };
+
+            var conditions = new List<ConditionApiModel> { condition };
+
+            return new RuleApiModel()
+            {
+                Name = calculation + " Test Rule",
+                Description = "Test Description",
+                GroupId = DEFAULT_CHILLERS_GROUP_ID,
+                Severity = "Info",
+                Enabled = true,
+                Calculation = calculation,
+                TimePeriod = timePeriod,
+                Conditions = conditions
+            };
+        }
+
+        /// <summary>
+        /// Returns true if the default chiller device group has been created by seed data.
+        /// Retries with a 10 sec timer if seed data in config service has not yet
+        /// created the device groups. Returns false after SEED_DATA_RETRY_COUNT failed attempts.
+        /// </summary>
+        private bool ValidChillerGroup()
```

Comment:
You can use [**this**](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet-tests/blob/master/Helper/SeedData.cs) helper method instead of creating a new one. 
Please modify the above method by adding an argument to replace the URL string in line 17.